source/pdo_sqlsrv/pdo_dbh.cpp

```diff
@@ -98,6 +102,24 @@ struct pdo_txn_isolation_conn_attr_func
     static void func( connection_option const* /*option*/, zval* value_z, sqlsrv_conn* conn, std::string& /*conn_str*/ TSRMLS_DC );
 };
 
+#ifdef _WIN32
+struct pdo_int_conn_str_func {
+
+    static void func( connection_option const* option, zval* value, sqlsrv_conn* /*conn*/, std::string& conn_str TSRMLS_DC )
+    {
+        TSRMLS_C;
+        SQLSRV_ASSERT( Z_TYPE_P( value ) == IS_STRING, "Wrong zval type for this keyword" ) 
```

Comment:
In conn.cpp, your assert message is "An integer is expected for this keyword", then should you change this one to "An string is expected for this keyword" just to be consistent?